libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -402,6 +402,50 @@ void AP_Mount::handle_gimbal_manager_set_attitude(const mavlink_message_t &msg){
     }
 }
 
+void AP_Mount::handle_command_gimbal_manager_set_pitchyaw(const mavlink_message_t &msg){
+    mavlink_gimbal_manager_set_pitchyaw_t packet;
+    mavlink_msg_gimbal_manager_set_pitchyaw_decode(&msg,&packet);
+
+    // check gimbal device id.  0 is primary, 1 is 1st gimbal, 2 is 2nd gimbal, etc
+    if (packet.gimbal_device_id > AP_MOUNT_MAX_INSTANCES) {
+        return;
+    }
+    const uint8_t gimbal_instance = (packet.gimbal_device_id < 1) ? get_primary_instance() : (uint8_t)packet.gimbal_device_id - 1;
```

Comment:
PeterB has added a new pattern for this since this PR was created.  Please have a peek at handle_command_do_gimbal_manager_pitchyaw() and you'll see it uses "get_primary()" and "get_instance()" so we should change this to use the same pattern.